app/utils/__init__.py

```
@@ -0,0 +1,20 @@
+    IDCounter
+)
+from .json_handler import JsonFileHandler
+
+__all__ = [
+    "generate_user_id",
+    "generate_post_id",
+    "generate_comment_id",
+    "get_id_counter",
+    "IDCounter",
+    "JsonFileHandler"
+]
```

Ruff (0.14.11)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)